cron/hourly/reconcile-transactions.js

```diff
@@ -48,6 +50,32 @@ async function reconcileConnectedAccount(connectedAccount) {
           'approvals',
         );
 
+        const collectiveId = card.collectiveId;
+        const collective = models.Collective.findOne({
+          where: {
+            id: collectiveId,
+          },
+        });
```

Comment:
:ice_cream: 

```suggestion
        const collective = await models.Collective.findByPk(collectiveId);
```

or 

```suggestion
        const collective = await card.getCollective();
```